web3/_utils/module_testing/module_testing_utils.py

```diff
@@ -106,7 +106,6 @@ def assert_contains_log(
     emitter_contract_address: ChecksumAddress,
     txn_hash_with_log: HexStr,
 ) -> None:
-    assert len(result) == 1
```

Comment:
Removing this may cause an index out of range when retrieving the first result. Not sure of the intent here but it's fine with me if we can assume there's a result and potentially have more than one.